src/model/liverange.js

```diff
@@ -168,6 +169,16 @@ function transform( operation ) {
 	}
 }
 
+/**
+ * Checks whether given operation changes something inside the range (even if it does not change boundaries).
+ *
+ * @ignore
```

Comment:
Usually we use `//` comments instead of `@ignore`.

Reply:
I changed all `@ignore` occurrences in the engine.